### PR TITLE
add eigenDA client protos

### PR DESCRIPTION
**Pull request by juanbono**
This PR adds the generated modules for the eigenDA protobuffs.

Auth will be implemented in another PR.

### CODE REVIEW

File: core/node/eigenda_proxy/proto/common/common.proto [hunk 1]
```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package common;
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
```
Comment (gianbelinche) [inline core/node/eigenda_proxy/proto/common/common.proto:3]: Do we need this go package?
Same with other files

File: core/node/eigenda_proxy/Cargo.toml [hunk 2]
```diff
@@ -14,3 +14,9 @@ anyhow.workspace = true
 axum.workspace = true
 tokio.workspace = true
 tracing.workspace = true
+prost = "0.13.1"
```
Comment (juan518munoz) [inline core/node/eigenda_proxy/Cargo.toml:17]: `prost` is available as a `workspace` dependency.

File: core/node/eigenda_proxy/proto/common/common.proto (hunk 1 above)
Comment (juanbono) [reply core/node/eigenda_proxy/proto/common/common.proto:3]: I added the proto files just to know where the compiled code comes from.

File: core/node/eigenda_proxy/Cargo.toml (hunk 2 above)
Comment (juanbono) [reply core/node/eigenda_proxy/Cargo.toml:19]: the prost version used workspace wide is older than the one required for tonic. I added a comment explaining that

**Review by juan518munoz — APPROVED** [review]

**Review by gianbelinche — APPROVED** [review]

**Pull request merged**